include/swift/AST/IRGenOptions.h

```diff
@@ -91,6 +94,9 @@ class IRGenOptions {
   /// Which sanitizer is turned on.
   SanitizerKind Sanitize : 2;
 
+  /// Which sanitizer coverage is turned on.
+  llvm::SanitizerCoverageOptions SanitizeCoverage;
```

Comment:
Oops, nitpick: please move this out of the bitfields, so it doesn't stop them from getting merged.